identity-verification/module_2/2.2 Onboarding IDV API Example.ipynb

```diff
@@ -545,7 +552,7 @@
    "source": [
     "## Clean up\n",
     "----\n",
-    "Here we simply need to delete our collections \n"
+    "As part of our cleanup, we can a delete our two collections. This will delete the collections and all the face vectors contained within.\n"
```

Comment:
```suggestion
    "As part of our cleanup, we can delete our two collections. This will delete the collections and all the face vectors contained within.\n"
```